test/models/mnist/validate_mnist.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+
+
+echo "**************************************************************************"
+echo "Run MNIST ADAM OPTIMIZER on NGRAPH"
+echo "**************************************************************************"
+python mnist_deep_simplified.py --train_loop_count=20
```

Comment:
Are we comparing against known values of accuracy etc? Or Is this script meant to be a one-stop runme for running the models. If we want a test, we can wrap this in a pytest

Reply:
one-stop runme for running the models with and without nGraph. I will open another PR for testing.